include/generate_os_token.py

```diff
@@ -0,0 +1,21 @@
+from keystoneauth1 import session, exceptions
```

Comment:
this quite fenix specific:-) i would not included here .. you also exposed the client secret

Reply:
That is indeed true, I copied the secrets from the CSCS repo. The file is deleted.